ethereum/core/src/main/java/org/hyperledger/besu/ethereum/BlockValidator.java

```diff
@@ -27,15 +27,18 @@ public interface BlockValidator {
   class Result {
     public final Optional<BlockProcessingOutputs> blockProcessingOutputs;
     public final Optional<String> errorMessage;
+    public final boolean isValid;
```

Comment:
where does this get used?

Reply:
in [RollupMergeCoordinator:L98](https://github.com/hyperledger/besu/blob/b52f2e1d6662eb7f06788973b11c53b1117f4754/consensus/merge/src/main/java/org/hyperledger/besu/consensus/rollup/blockcreation/RollupMergeCoordinator.java#L98) class